doc/classes/EditorInterface.xml

```diff
@@ -159,9 +159,10 @@
 		<method name="select_file">
 			<return type="void">
 			</return>
-			<argument index="0" name="file" type="String">
+			<argument index="0" name="file_path" type="String">
```

Comment:
Did you use the class generation script to change that? You shouldn't change the XML data manually as it's automatically pulled from the source code.